src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        ColorSupplier randcolor = new ColorSupplier();
+        Random num = new Random();
+        return String.valueOf(new Ball(randcolor.getRandomColor(), num.nextInt(100)));

```

Comment:
100 should be made a constant field (private static final field with name in UPPER_CASE)
https://www.javatpoint.com/java-constant